models/transactions.go

```diff
@@ -75,7 +75,7 @@ func (tdb *TransactionDB) Transact(t *Transaction) bool {
 
 	// Rollback transaction on any failures
 	defer func() {
```

Comment:
The `defer` is used to rollback for any errors inside the function.  The `err` is declared in the top of the function, so it should be available inside the `defer`.